pkg/tangle/tangle.go

```diff
@@ -153,32 +150,12 @@ func New(
 		receiveBlockWorkerCount:          2 * runtime.NumCPU(),
 		receiveBlockQueueSize:            10000,
 		futureConeSolidifierWorkerCount:  1, // must be one, so there are no parallel solidifications of the same cone
-		futureConeSolidifierQueueSize:    10000,
 		processValidMilestoneWorkerCount: 1, // must be one, so there are no parallel validations
-		processValidMilestoneQueueSize:   1000,
 		milestoneSolidifierWorkerCount:   2, // must be two, so a new request can abort another, in case it is an older milestone
```

Comment:
Hopefully it is no problem for the solidifier logic if we don't have the workerpool with exactly the 2 possible "slots" and not use the "TrySubmit" (which would drop more than 2 pending requests to solidify)